test/unit/pcv/lido/EthLidoPCVDeposit.test.ts

```diff
@@ -57,6 +58,12 @@ describe('EthLidoPCVDeposit', function () {
       await ethers.getContractFactory('EthLidoPCVDeposit')
     ).deploy(
       this.core.address,
+      {
```

Comment:
Getting some errors on typing for the oracle params, are you sure this is correct? My types show that the second param here should be a string.

Reply:
you may need to recompile, I had this problem while developing, the old types were used and I got a lot of warnings in my editor. I have added a structure for oracle params in the constructor, but the old 2nd parameter was a string (now the 3rd parameter is a string, and 2nd is a structure)

Reply:
one more reason why I don't like that we add a compilation layer on top of js btw